FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -897,6 +898,29 @@ extension ComposeViewController: UIDocumentPickerDelegate {
     }
 }
 
+// MARK: - PHPickerViewControllerDelegate
+extension ComposeViewController: PHPickerViewControllerDelegate {
+    nonisolated func picker(_ picker: PHPickerViewController, didFinishPicking results: [PHPickerResult]) {
+        Task {
+            await picker.dismiss(animated: true)
+            let itemProvider = results.first?.itemProvider
+            itemProvider?.loadFileRepresentation(
+                forTypeIdentifier: "public.item",
+                completionHandler: { [weak self] url, _ in
+                    guard let self = self else { return }
+                    DispatchQueue.main.async {
+                        if let url = url, let composeMessageAttachment = ComposeMessageAttachment(fileURL: url) {
+                            self.appendAttachmentIfAllowed(composeMessageAttachment)
+                            self.node.reloadSections(IndexSet(integer: 2), with: .automatic)
+                        } else {
+                            self.showAlert(message: "files_picking_photos_error_message".localized)
+                        }
+                    }
+                })
+        }
+    }
+}
+
 // MARK: - UIImagePickerControllerDelegate & UINavigationControllerDelegate
 extension ComposeViewController: UIImagePickerControllerDelegate, UINavigationControllerDelegate {
     func imagePickerController(
```

Comment:
We don't need `ComposeViewController` conform to `UIImagePickerControllerDelegate` and `UINavigationControllerDelegate`, as image picker is replaced with `PHPhotoPicker`.
This delegate method `imagePickerController(...` isn't used anymore too.

Reply:
We do, because we need to let user to take a photo from app.

Reply:
Got it, I missed that we still need ability to take photo, not only select existing one from the library.